packages/cron/README.md

```diff
@@ -0,0 +1,105 @@
+# Create a Cron Job with Functionless
+
+In this example, we'll create a Cron job.
+
+The Cron Job will use [AWS EventBridge's](https://aws.amazon.com/eventbridge/) scheduled rules to trigger a job in a lambda [Lambda Functions](https://docs.aws.amazon.com/lambda/latest/dg/welcome.html).
+
+Then we'll deploy and test the application.
+
+To see the completely application, go [here](https://github.com/functionless/functionless-samples/packages/cron).
+
+## Create an new app
+
+> yarn create functionless
+
+For a more detailed guide, see: https://functionless.org/docs/getting-started/setup
+
+## Create the cron job
+
+Replace your `app.ts` with (the imports will be useful later):
+
+```ts
+import { App, Duration, Stack } from "aws-cdk-lib";
+import { Schedule } from "aws-cdk-lib/aws-events";
+import { EventBus, Function } from "functionless";
+
+const app = new App();
+const stack = new Stack(app, "functionlessCron");
+
+EventBus.schedule(stack, "cron", Schedule.rate(Duration.minutes(1)));
+```
+
+This create a rule which executes any targets every minutes. Lets give it something to do.
```

Comment:
Consider:

"An EventBus Scheduled Rule emits a triggering event at a rate specified by the duration argument. In this example, we'll emit an event every minute. These triggers can be used to trigger downstream integrations (e.g. AWS Lambda, AWS Step Functions, etc.)."

I would also probably kill the next section and just introduce what you will be triggering.